lib/internal/errors.js

```diff
@@ -433,6 +433,8 @@ E('ERR_MODULE_RESOLUTION_LEGACY', '%s not found by import in %s.' +
 E('ERR_MULTIPLE_CALLBACK', 'Callback called multiple times');
 E('ERR_NAPI_CONS_FUNCTION', 'Constructor must be a function');
 E('ERR_NAPI_CONS_PROTOTYPE_OBJECT', 'Constructor.prototype must be an object');
+E('ERR_NAPI_INVALID_DATAVIEW_ARGS',
+  'Invalid arguments were passed to napi_create_dataview()');
```

Comment:
The message here should match the message in src/node_api.cc.  There will be a better way to handle errors on the native side but for now I think we want to make the messages the same.

Reply:
I should have suggested that in the original comment.

Reply:
Thank you for explanation.
I've addressed your comment in my latest patch.